README.md

```diff
@@ -196,13 +197,13 @@ the Equinix Metal CCM enables BGP on the project and nodes, assigns an EIP for e
 `Service`, and adds annotations to the nodes. These annotations are configured to be consumable
 by kube-vip.
 
-To enable it, set the configuration `PACKET_LB` or config `packetLB` to:
+To enable it, set the configuration `METAL_LB` or config `metalLB` to:
```

Comment:
This name is a little confusing because `metallb` is one of the potential values, I wonder if we should do something about it in this rename.

```
METAL_LB=metallb://
METAL_LB=kube-vip://
METAL_LB=
```

fwiw, I can live with it.

Reply:
Yeah, that bothered me as well, but I couldn't think of anything better. `EQUINIXMETAL_LB` and `EQUINIX_METAL_LB` or just too much of a mouthful. `PACKET_LB` was great, but we obviously aren't using that. Let us leave it.